src/operator/tensor/elemwise_unary_op.cc

```diff
@@ -426,7 +426,7 @@ MXNET_OPERATOR_REGISTER_BINARY(_backward_arctan)
 
 // degrees
 MXNET_OPERATOR_REGISTER_UNARY(degrees)
-.describe(R"code(Convert angles from radians to degrees.
+.describe(R"code(Returns element-wise angles in degrees converted from radian angles of the input.
```

Comment:
This is written confusingly, no need to mention "element-wise angles". Just say: 

"Converts each element of the input array from radians to degrees."